### DISCUSSION TITLE
Online bot detector detects it

### DISCUSSION THREAD

**Discussion by ibrahimmus** [apify/fingerprint-suite#319] [General]
**Describe the bug**
https://fingerprint.com/products/bot-detection/ Detects the usage of automation script. 

**To Reproduce**
I am using the puppeteer example you have provided in readme. Only thing i changed is page.goto part where I put the link: https://fingerprint.com/products/bot-detection/.

**Expected behavior**
(https://fingerprint.com/products/bot-detection/) should not detect the usage of automated environment. 

**System information:**
 - OS: Windows
 - Node.js version v18.18.0

**Comment by barjin** [comment]
We don't have enough information about the fingerprint.com project and its internals. We are currently discussing the future of this project, as we keep noticing its degrading performance against existing antibot solutions.

A lot of issues in this repo refer to some fingerprinting benchmarks that don't necessarily reflect real world antibot services. Please, report only cases where you were trying to access (scrape, automate) an actual website (Netflix, Amazon, Google Maps...) and it didn't work. While optimizing the tool for benchmarks might help with real-world usage, it also draws our attention and manpower to artificial problems.

**Comment by ibrahimmus** [comment]
Understand, thanks for the response. This is good stuff so hope that you will be actively maintaining it.
